src/TestWrapper.js

```diff
@@ -1,6 +1,6 @@
 export default class TestWrapper {
   isPresent () {
-    return this.wrapper.length > 0
+    return this.wrapper.children().length > 0
```

Comment:
This is probably not be what we want, I was confused initially that the behaviour of `isPresent` & `isEmpty` was different.